### PR TITLE
docs: update docs for quote character check

### PR DESCRIPTION
**Pull request by sebastiantoh**
This PR updates the documentation based on the changes made here: https://github.com/dotenv-linter/dotenv-linter/pull/349

### CODE REVIEW

File: docs/checks/quote_character.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Quote character
 
-Detects if a value contains quote characters (`'` / `"`):
+Detects if a value contains quote characters (`'` / `"`) but skips if the value contains whitespaces:
```
Comment (mgrachev) [inline docs/checks/quote_character.md]:
```suggestion
Detects if a value contains quote characters (`'` / `"`) but skips if the value contains whitespaces or newline characters:
```

File: docs/checks/quote_character.md [hunk 2]
```diff
@@ -14,4 +14,10 @@ FOO='B"AR'
 
 ✅ Correct
 FOO=BAR
+
+✅ Correct
+FOO="BAR BAR"
+
+✅ Correct
+FOO='BAR BAR'
```
Comment (mgrachev) [inline docs/checks/quote_character.md:22]:
```suggestion
FOO='BAR BAR'

✅ Correct
FOO="BAR\nBAR"
```

**Review by mgrachev — COMMENTED** [review]
@sebastiantoh Thank you for your help 🙂

I have left some comments. Please take a look at them 👀

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@sebastiantoh Please resolve merge conflict 🙏

**Comment by sebastiantoh** [comment]
Closing in favour of #23 (accidentally deleted my fork earlier - so no longer have access to the branch for this PR)

**Pull request closed without merging**